how-to/networking/install-dnssec.md

```diff
@@ -0,0 +1,190 @@
+(install-dnssec)=
+# Installing DNS Security Extensions (DNSSEC)
+
+DNSSEC is a set of security extensions to DNS which allow DNS data to be verified for authenticity and integrity.
+
+This how-to will show how to enable DNSSEC for an existing zone in your bind9 DNS server deployment.
+
+## Starting point
+The starting point for this how-to is an existing bind9 DNS server deployed with an authoritative zone. For details on how to deploy bind9 in this fashion, please see the [DNS How-To](https://github.com/panlinux/ubuntu-server-documentation/blob/bind9-dnssec/how-to/networking/install-dns.md). One key difference from that guide, however, is that we need the zone file to be in a directory where bind9 can write to, like `/var/lib/bind`, instead of `/etc/bind`.
+
+Nevertheless, here is a quick set of steps to reach that state for an example domain called `example.internal`.
+
+First, install the bind9 package:
+
+    sudo apt install bind9 -y
+
+Edit `/etc/bind/named.conf.local` and add this *zone* block:
+
+    zone "example.internal" {
+        type master;
+        file "/var/lib/bind/db.example.internal";
+    };
+
+Create the file `/var/lib/bind/db.example.internal` with these contents:
+
+    $TTL 86400      ; 1 day
+    example.internal.       IN SOA  example.internal. root.example.internal. (
+                                    1          ; serial
+                                    43200      ; refresh (12 hours)
+                                    900        ; retry (15 minutes)
+                                    1814400    ; expire (3 weeks)
+                                    7200       ; minimum (2 hours)
+                                    )
+    example.internal.       IN  NS      ns.example.internal.
+    ns                      IN  A       192.168.1.10
+    noble                   IN  A       192.168.1.11
+
+Restart the service:
+
+    sudo systemctl restart named
+
+Check if the service can resolve the name `noble.example.internal`:
+
+    $ dig @127.0.0.1 +short noble.example.internal
+    192.168.1.11
+
+## Enabling DNSSEC
+Enabling DNSSEC for a zone involves multiple steps. Thankfully for us, the bind9 DNS server takes care of all of them by default, automatically, leaving very little for us to do. Converting a zone in this way means at least generating new keys, and signing all the resource records of the zone. But that is only "day 1": such a zone must be maintained properly. Keys must be rotated, expiration dates must be set, etc. The current versions of bind9 take care of that with a dnssec policy, and of course there is a default one that can be used from the get go.
```

Comment:
Will the user know which keys we mean when we say "generating new keys" and "rotating keys"?

Reply:
The idea here is to just show how much more complicated things could be, how there are many important details pertaining DNSSEC, but that these are taken care of by the server via the dnssec policy.